### DISCUSSION TITLE
Welcome to B450M-MORTAR-MAX-GT740 Discussions!

### DISCUSSION THREAD

**Discussion by jacobax** [jacobax/B450M-MORTAR-MAX-GT740#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

有问题尽管提出讨论，但很可能你懂得比我多
黑苹果需要一点动手能力和耐心，更重要的是愿意且主动学习

**Comment by zadedouxing** [comment]
非常nb克拉斯 感谢分享！
